.pre-commit-config.yaml

```diff
@@ -184,9 +184,12 @@ repos:
         exclude: tests/metagen/typegraphs/sample/rs/client.rs
       - id: clippy
         args:
+          - "--workspace"
           - "--locked"
           - "--all-features"
           - "--all-targets"
+          - "--exclude"
+          - "client_rs_static"
```

Comment:
Note, we wan't to avoid triggering clippy in the user's workspace after the code is generated. Might be better to disable rules on the module directly if you need to instead of completely excluding it.